ksql-engine/src/test/java/io/confluent/ksql/services/KafkaTopicClientImplTest.java

```diff
@@ -290,12 +335,33 @@ public void shouldThrowTopicDeletionDisabledException()
     kafkaTopicClient.deleteTopics(Collections.singletonList(topicName1));
   }
 
+  @Test
+  public void shouldFailToDeleteOnThrowTopicAuthorizationException()
```

Comment:
nit:
```suggestion
  public void shouldFailToDeleteOnTopicAuthorizationException()
```
to be consistent with `shouldFailToDescribeOnTopicAuthorizationException()` above.

Same comment for the test above as well.